example_docs/docs/conf.py

```diff
@@ -27,6 +27,7 @@
     "source_repository": "https://github.com/Qiskit/qiskit_sphinx_theme/",
     "source_branch": "main",
     "source_directory": "example_docs/docs/",
+    "sidebar_qiskit_ecosystem_member": True,
```

Comment:
To discourage false positives, I'd set this to False here and instead, add a comment saying to change this to True for an ecosystem project.

Reply:
My only concern is we'd lose the visual regression test. We can do something like this, though:

```python

# Members of the Qiskit ecosystem should set `sidebar_qiskit_ecosystem_member` explicitly in `html_theme_options`.
# This convoluted code is only set up this way to discourage projects copying
# this config file from unintentionally configuring this option.
if project == "example docs":
    html_theme_options["sidebar_qiskit_ecosystem_member"] = True
```

Reply:
I'm okay with that